lua/lualine/utils/utils.lua

```diff
@@ -54,4 +54,8 @@ function M.clear_highlights()
   end
 end
 
+function M.get_short_cwd()
```

Comment:
This probably can be inlined in nerdtree extention file. Since Other extentions are directly using nerdtree extention, I don't see what else will use it . Though leaving it in utils isn't much of an issue either :)

Reply:
Yeah I will move it to `nerdtree.lua` since nothing else is using it 👍